client/src/components/common/Navbar/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button } from 'antd';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faShoppingCart } from '@fortawesome/free-solid-svg-icons';
+import * as ROUTES from '../../../constants/routes';
+import navLogo from '../../../assets/images/berwaz.png';
+import './style.css';
+
+const NavbarComponent = () => {
+  return (
+    <div className="main-navbar-container">
+      <div className="navbar-container">
+        <div className="nav-left">
+          <Link to={ROUTES.CART_PAGE}>
+            <FontAwesomeIcon
+              className="cart-icon"
+              icon={faShoppingCart}
+              size="lg"
+            />
+          </Link>
+          <Link to={ROUTES.SIGNUP_PAGE} className="nav-link">
+            مستخدم جديد
+          </Link>
+          <Link to={ROUTES.LOGIN_PAGE}>
+            <Button className="nav-login-btn">تسجيل الدخول</Button>
```

Comment:
Don't use a button as a link. Instead, use a link styled as a button.
try something like this
```jsx
<Link to={ROUTES.LOGIN_PAGE} className="btn btn-primary">تسجيل الدخول</Link>
```